FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -101,26 +108,54 @@ - (instancetype)initWithDatabaseManager:(RCNConfigDBManager *)manager
                                                                   bundleID:_bundleIdentifier
                                                                  namespace:_FIRNamespace];
     _isFetchInProgress = NO;
+
+    // Check if app group is set. If so, use it to communicate the latest fetch time.
+    if (options.appGroupID) {
+      _sharedUserDefaultsSuiteName = options.appGroupID;
+      /// Instantiate the shared instance.
+      _sharedUserDefaults = [self sharedUserDefaults];
+    }
   }
   return self;
 }
 
+- (NSUserDefaults *)sharedUserDefaults {
```

Comment:
@ryanwilson Should this be GULUserDefaults?

Reply:
FWIW, we use another NSUserDefaults suite currently in RC (not  GULUserDefaults).

Reply:
It should be `GULUserDefaults`. It uses the same underlying storage so there's no worries there, it's compatible with everything else, but when it's written to it doesn't have the bug we've seen on iOS 11 where an Apple framework tries to update UI on the thread it's fired from.

One side-effect is calling `sync` can take up to 1 second, although we need to re-evaluate that.

Reply:
NSUserDefaults here will be called only from RC serial queue (no UI thread). Would it still be required to GULUserDefaults here?